recipes/gatk4/meta.yaml

```diff
@@ -11,13 +11,17 @@ source:
 
 build:
   noarch: generic
-  number: 0
+  number: 1
 
 requirements:
   host:
   run:
     - openjdk >=8,<9
     - python
+    - r-gplots
```

Comment:
can you please add r-base as well

Reply:
Thanks!
Done
I though that it was implicite with any r package

Reply:
It is impplicit, but conda has some magic that is done if you specify the main scripting language.